content/_data/changelogs/weekly.yml

```diff
@@ -5291,6 +5291,50 @@
     authors:
       - jglick
 
+- version: '2.204'
+  date: 2019-11-10
+  changes:
+
+  # RFEs
+
+  - type: rfe
+    pull: 4273
+    issue: 59679
+    message: |-
+      Plugin manager descriptions always link to the plugins site instead of the Jenkins wiki.
+    authors:
+      - zbynek
+  - type: rfe
+    pull: 4341
+    message: |-
+      Increase the number of datapoints recorded for multistage time series graphs such as those used for load statistics
+    authors:
+      - daniel-beck
+  - type: rfe
+    pull: 4340
+    references:
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-3.36
+        title: Remoting release notes
+    message: |-
+      Update Remoting from 3.35 to 3.36 to add new command line options "-help" and "-version".
+    authors:
+      - jeffret-b
+
+  # Developer-facing changes (APIs, extensions, etc.)
+
+  - type: bug
+    pull: 4311
+    issue: 59886
+    message: |-
+      Developer: Strengthen the queue to prevent canTake() and canRun() implementations in NodeProperty and QueueTaskDispatcher extensions from hanging the queue
```

Comment:
AFAICT actually a robustness improvement with user-visible behavior changes when plugins go bad.

Quick reminder: https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/_STYLEGUIDE.adoc#labels

Reply:
Ah, when I classified it as "Developer:", I was assuming that it would be most relevant to developers rather than to users.  You are noting that a user is much more interested when they are harmed by a plugin bug that causes the queue to hang.

I had read the style guide in detail before preparing the pull request but didn't consider the user impact as clearly as I should have.  Thanks!

Reply:
>  You are noting that a user is much more interested when they are harmed by a plugin bug that causes the queue to hang.

Exactly 👍 This change, AFAIUI, is expected to slightly improve Jenkins for users if they use unstable plugins. This is similar to other changes in the changelog that mention "robustness", e.g. in https://jenkins.io/changelog/#v2.187 or https://jenkins.io/changelog/#v2.181.

"Developer:" means users simply won't care. This typically means new APIs for plugins, or tooling updates to the pom.xml (although plugin dev vs. core dev is not well defined, so it's a best guess kind of thing).

"Internal:" means neither users nor (plugin) developers will care, e.g. our recent replacement of one base64 implementation in core with another, or substantial internal code cleanup/refactoring. It's still a significant enough change to note it.

Obviously these are categories I made up without gathering much input (if any, it's been too long ago and I don't remember), they just made sense to me. Whether you decide to keep them and/or their current definitions, is up to you.